packages/grafana-data/src/dataframe/index.ts

```diff
@@ -6,6 +6,5 @@ export * from './processDataFrame';
 export * from './dimensions';
 export * from './ArrayDataFrame';
 export * from './DataFrameJSON';
-export { StreamingDataFrame, StreamingFrameOptions, StreamingFrameAction } from './StreamingDataFrame';
```

Comment:
@mckn -- this removes `@alpha` interfaces from the package.

I *think* it is OK because the were not really useable outside the live query runner anyway.  Any other use would just be `DataFrameJSON` objects

Reply:
Lets keep an eye on this -- we could also put back the original implementations as @deprecated if this change causes any problemns

Reply:
Okay, I this is is great that we are moving things away from `@grafana/data` that doesn't make sense to use outside of grafana core. 

I know that we have a couple of plugins that are starting to use this functionality e.g. the kafka plugin but I you are certain that they are not using this then I think it is fine to remove. I guess that since we are removing this we can't add any migration guides because there are nothing to migrate to. Right?

Reply:
We will have to check how the kafka plugin is actually written to know for sure -- if done as backend, then the streaming behavior is all managed transparently by `DatasourceWithBackend`